vitals/Main.java

```diff
@@ -1,23 +1,42 @@
 package vitals;
 
+import vitals.interfaces.VitalsInterface;
+import vitals.test.VitalsTest;
+
 public class Main {
-    static boolean batteryIsOk(float temperature, float soc, float chargeRate) {
-        if(temperature < 0 || temperature > 45) {
-            System.out.println("Temperature is out of range!");
-            return false;
-        } else if(soc < 20 || soc > 80) {
-            System.out.println("State of Charge is out of range!");
-            return false;
-        } else if(chargeRate > 0.8) {
-            System.out.println("Charge Rate is out of range!");
-            return false;
-        }
-        return true;
-    }
-
-    public static void main(String[] args) {
-        assert(batteryIsOk(25, 70, 0.7f) == true);
-        assert(batteryIsOk(50, 85, 0.0f) == false);
-        System.out.println("Some more tests needed");
-    }
+
+  public static void main(final String[] args) {
```

Comment:
batteryIsOk() is an API that provides convenient way to user to check overall batteryhealth as this API abstracts all the checks within.
By excluding such facade API, the user is now forced to check each parameter and then decide battery health.
is there a reason why this API was excluded?